lib/jackalope/session.ex

```diff
@@ -43,6 +43,11 @@ defmodule Jackalope.Session do
     GenServer.cast(__MODULE__, {:report_connection_status, status})
   end
 
+  @spec connection_query() :: :online | :offline
+  def connection_query() do
```

Comment:
```suggestion
  def connection_status() do
```
Can we have this match the key of the state it is returning?

Reply:
Just making sure I understand, we want to match the key in the args?

Reply:
updated